README.md

```diff
@@ -883,8 +883,7 @@ Other Style Guides
     ```javascript
     // bad
     [1, 2, 3].map(number => {
-      const nextNumber = number + 1;
```

Comment:
Right. The intention is to call out the confusion that definitely happens between implicit and explicit return in arrow functions. In this case, the bad example is because it's forgetting the `return` keyword.